api/enterprise-api-ref/reset-password.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: default
+title: Reset password API
+published: true
+---
+
+## Request reset password link and token
+
+This call initiates the password reset process by sending a reset password
+link and token to the user's registered email address. User can click the link from the email
+and set a new password in the Mission Portal or invalidate the request.
+Every request has an expiration time equals to 48 hours.
+
+**URI:** https://hub.cfengine.com/api/auth/password/forgot/:username
+
+**Method:** POST
+
+**Example request (curl):**
+
+```console
+curl -X POST \
+  https://hub.cfengine.com/api/auth/password/forgot/admin
+```
+
+**Successful response example:**
+
+```
+HTTP 200 Ok
+
+Reset password email successfully sent.
+```
+
+**Responses:**
+
+| HTTP response code       | Description                                                                         |
+|--------------------------|-------------------------------------------------------------------------------------|
+| 200 OK                   | Reset password email successfully sent                                              |
```

Comment:
Is this API in the open then? I would expect it would be and in that case we would want to only return 200 OK and maybe 500 server errors so that we don't reveal information about users found or not found or time threshold algorithms.

Reply:
yes, changed accordingly to api